File: backend/api/Models/ServerEvents/ServerNotifiesClientsInRoomSomeoneHasJoinedRoom.cs
using Externalities.QueryModels;

namespace api.Models.ServerEvents;

public class ServerNotifiesClientsInRoomSomeoneHasJoinedRoom : BaseDto
{
    public int roomId { get; set; }

    public string? message { get; set; }
    public EndUser user { get; set; }
}

File: frontend/src/app/models/serverNotifiesClientsInRoomSomeoneHasJoinedRoom.ts
import {BaseDto} from "./baseDto";
import {EndUser} from "./entities";

export class ServerNotifiesClientsInRoomSomeoneHasJoinedRoom extends BaseDto<ServerNotifiesClientsInRoomSomeoneHasJoinedRoom> {
  user?: EndUser
  roomId?: number;
  message?: string;
}
